tests/test_providers/test_address.py

```diff
@@ -149,80 +149,53 @@ def test_calling_code(self, _address):
 
 
 class TestSeededAddress(object):
-    TIMES = 5
 
     @pytest.fixture
-    def _addresses(self, seed):
-        return Address(seed=seed), Address(seed=seed)
-
-    def test_street_number(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.street_number() == a2.street_number()
-
-    def test_latitude(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.latitude() == a2.latitude()
-
-    def test_longitude(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.longitude() == a2.longitude()
-
-    def test_coordinates(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.coordinates() == a2.coordinates()
-
-    def test_street_name(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.street_name() == a2.street_name()
-
-    def test_street_suffix(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.street_suffix() == a2.street_suffix()
-
-    def test_address(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.address() == a2.address()
-
-    def test_state(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.state() == a2.state()
-            assert a1.state(abbr=True) == a2.state(abbr=True)
-
-    def test_postal_code(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.postal_code() == a2.postal_code()
-
-    def test_country(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.country() == a2.country()
-
-    def test_country_iso(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.country_iso_code() == a2.country_iso_code()
-
-    def test_city(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.city() == a2.city()
-
-    def test_continent(self, _addresses):
-        a1, a2 = _addresses
-        for _ in range(self.TIMES):
-            assert a1.continent() == a2.continent()
-            assert a1.continent(code=True) == a2.continent(code=True)
```

Comment:
@lk-geimfari Check seeded provider methods with arguments also neccesary. Because there may be specific conditions when we pass arguments to method. When i wrote seed tests in first time there were many such situations.

Reply:
@duckyou I'll revert changes related to such cases.

Reply:
@duckyou I have revert changes. Also, we must think about doing library easily extensible. We must get rid of the hard-coded parts of the code and I hope to your help.

Reply:
I have created #392